File: 1672-richest-customer-wealth/1672-richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int n=accounts.size();
        int ans = -1;
        for(int i=0; i<n; i++){
            ans = max(ans, accumulate(accounts[i].begin(), accounts[i].end(), 0));
        }
        return ans;
    }
};

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int ans = 0;
        int n = accounts.length;
        
        for(int i=0; i<n; i++){
            int curr_row_sum = 0;
            int m = accounts[i].length;
            for(int j=0; j<m; j++){
                curr_row_sum += accounts[i][j];
            }
            ans = Math.max(curr_row_sum, ans);
        }
        return ans;
    }
}